util/util.go

```diff
@@ -92,3 +94,44 @@ func CheckKVFlag(c *cli.Context, flagName string) map[string]string {
 	}
 	return kv
 }
+
+// Table writes a table listing from an array of map[string]interface{}
+func Table(c *cli.Context, many []map[string]interface{}, keys []string) {
+	w := tabwriter.NewWriter(c.App.Writer, 0, 8, 1, '\t', 0)
+	// Write the header
+	fmt.Fprintln(w, strings.Join(keys, "\t"))
+
+	for _, m := range many {
+		writeMapEntry(w, m, keys)
+	}
+	w.Flush()
+}
+
+// MetaDataTable writes standardized metadata out
+// Solely a utility method that invokes MetaDataMapTable with structs.Map(i)
+func MetaDataTable(c *cli.Context, i interface{}, keys []string) {
+	m := structs.Map(i)
+	MetaDataMapTable(c, m, keys)
+}
+
+// MetaDataMapTable writes standardized metadata out
+func MetaDataMapTable(c *cli.Context, m map[string]interface{}, keys []string) {
+	w := tabwriter.NewWriter(c.App.Writer, 0, 8, 0, '\t', 0)
+
+	fmt.Fprintln(w, "PROPERTY\tVALUE")
+
+	for _, key := range keys {
+		val := fmt.Sprint(m[key])
+		fmt.Fprintf(w, "%s\t%s\n", key, strings.Replace(val, "\n", "\n\t", -1))
+	}
+	w.Flush()
+}
+
+// writeMapEntry writes a table entry from a map
+func writeMapEntry(w *tabwriter.Writer, m map[string]interface{}, keys []string) {
```

Comment:
It was when I had these broken up into one for a map interface and one for a slice of interfaces. Then I simplified this greatly. This could be brought back up into ListTable